app/components/dialogs/prioritize-build-modal.js

```diff
@@ -0,0 +1,104 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+import eventually from 'travis/utils/eventually';
+import { reads } from '@ember/object/computed';
+import { task } from 'ember-concurrency';
+import {
+  bindKeyboardShortcuts,
+  unbindKeyboardShortcuts
+} from 'ember-keyboard-shortcuts';
+
+export default Component.extend({
+  keyboardShortcuts: {
+    'esc': 'toggleConfirmationModal'
+  },
+  flashes: service(),
+
+  didInsertElement() {
+    this._super(...arguments);
+    bindKeyboardShortcuts(this);
+  },
+
+  willDestroyElement() {
+    this._super(...arguments);
+    unbindKeyboardShortcuts(this);
+  },
+  isOpen: false,
+  options: computed(() => {
+    let arr = [
+      {
+        key: 'private1',
+        displayValue: 'item1',
+        cancelRunningJobsVal: false,
+        description: 'Place build at the top of the queue',
+        modalText: 'Place build at top of the queue',
+      },
+      {
+        key: 'private2',
+        displayValue: 'item2',
+        cancelRunningJobsVal: true,
+        description: 'Place build at the top of the queue and cancel all running jobs',
+        modalText: 'Place build at the top of the queue and cancel all running jobs',
+      }
+    ];
+    return arr;
+  }),
```

Comment:
All this data is mainly presentation. I'd like to suggest moving it to template, as presentation is the template's concern. Also, as long as there are just two options currently, there's no need for iteration, just put them both one by one, set the `cancelRunningJobsVal` on change and calculate selection state by equality to this property, e.g. `selected={{eq cancelRunningJobsVal true}}` and `selected={{eq cancelRunningJobsVal false}}` accordingly. Does it make sense to you?

Reply:
Implemented as suggested.